.github/workflows/release.yaml

```diff
@@ -3,6 +3,7 @@
 # Apache License 2.0
 
 name: Release
+
```

Comment:
Unrelated change. It will be better to add such changes in separate commit. This makes it easy to revert the schedule change if needed.